frontend/src/store/store.test.js

```diff
@@ -232,39 +262,54 @@ describe('Vuex store', () => {
         expect(commit).toHaveBeenNthCalledWith(6, 'newsletterState', true)
       })
 
+      it('commits gmsAllowed', () => {
+        login({ commit, state }, commitedData)
+        expect(commit).toHaveBeenNthCalledWith(7, 'gmsAllowed', true)
+      })
+
+      it('commits gmsPublishName', () => {
+        login({ commit, state }, commitedData)
+        expect(commit).toHaveBeenNthCalledWith(8, 'gmsPublishName', 'GMS_PUBLISH_NAME_FULL')
+      })
+
+      it('commits gmsPublishLocation', () => {
+        login({ commit, state }, commitedData)
+        expect(commit).toHaveBeenNthCalledWith(9, 'gmsPublishLocation', 'GMS_LOCATION_TYPE_EXACT')
+      })
+
       it('commits hasElopage', () => {
         login({ commit, state }, commitedData)
-        expect(commit).toHaveBeenNthCalledWith(7, 'hasElopage', false)
+        expect(commit).toHaveBeenNthCalledWith(10, 'hasElopage', false)
       })
 
       it('commits publisherId', () => {
         login({ commit, state }, commitedData)
-        expect(commit).toHaveBeenNthCalledWith(8, 'publisherId', 1234)
+        expect(commit).toHaveBeenNthCalledWith(11, 'publisherId', 1234)
       })
 
       it('commits roles', () => {
         login({ commit, state }, commitedData)
-        expect(commit).toHaveBeenNthCalledWith(9, 'roles', ['admin'])
+        expect(commit).toHaveBeenNthCalledWith(12, 'roles', ['admin'])
       })
 
       it('commits hideAmountGDD', () => {
         login({ commit, state }, commitedData)
-        expect(commit).toHaveBeenNthCalledWith(10, 'hideAmountGDD', false)
+        expect(commit).toHaveBeenNthCalledWith(13, 'hideAmountGDD', false)
       })
 
       it('commits hideAmountGDT', () => {
         login({ commit, state }, commitedData)
-        expect(commit).toHaveBeenNthCalledWith(11, 'hideAmountGDT', true)
+        expect(commit).toHaveBeenNthCalledWith(14, 'hideAmountGDT', true)
       })
     })
 
     describe('logout', () => {
       const commit = jest.fn()
       const state = {}
 
-      it('calls twelve commits', () => {
+      it('calls seventeen commits', () => {
```

Comment:
```suggestion
      it('calls fifteen commits', () => {
```
the same here again `twelve commits` plus three new commits => 15

Reply:
Vorher waren es 14 + 3 = 17.

Reply:
ok